tests/unit/test_command_install.py

```diff
@@ -9,39 +10,90 @@
     decide_user_install,
     warn_deprecated_install_options,
 )
+from pip._internal.exceptions import CommandError, InstallationError
 from pip._internal.req.req_install import InstallRequirement
 
+ENABLE_USER_SITE = 'site.ENABLE_USER_SITE'
+WRITABLE = 'pip._internal.commands.install.test_writable_dir'
+ISDIR = 'pip._internal.commands.install.os.path.isdir'
+EXISTS = 'pip._internal.commands.install.os.path.exists'
 
+
+def false(*args, **kwargs):
+    """Return False."""
+    return False
+
+
+def true(*args, **kwargs):
+    """Return True."""
+    return True
+
+
+# virtualenv_no_global is tested by functional test
+@patch('pip._internal.commands.install.virtualenv_no_global', false)
```

Comment:
pytest-way is to do this via a fixture.

Reply:
How can I use a module-wise patch fixture though?